editoast/src/views/timetable.rs

```diff
@@ -449,21 +458,49 @@ impl Conflict {
 }
 
 #[derive(Debug, Clone, PartialEq, Deserialize, Serialize, ToSchema)]
-struct PacedTrainOccurrenceId {
-    paced_train_id: i64,
-    index: u64,
+#[serde(untagged)]
+enum ConflictPacedTrainId {
+    Occurence {
+        paced_train_id: i64,
+        index: u64,
+    },
+    Exception {
+        paced_train_id: i64,
+        exception_key: String,
+        index: Option<u64>,
+    },
 }
 
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, PartialEq)]
+/// This ID is used to identify paced train occurrences and exceptions when sending them to the core API for conflict detection.
 enum TrainId {
+    /// Id format `{id}`
     TrainSchedule(i64),
-    PacedTrainOccurrence { paced_train_id: i64, index: u64 },
+    /// Id format `{paced_train_id}#{index}`
+    PacedTrainOccurrence {
+        paced_train_id: i64,
+        index: u64,
+    },
+    /// Id format `{paced_train_id}@{exception_key}@{index}`
+    PacedTrainException {
+        paced_train_id: i64,
+        exception_key: String,
+        index: Option<u64>,
+    },
```

Comment:
With the modification I propose above, you could just do the following.

```suggestion
    /// Id format `{paced_train_id}#{index}`
    PacedTrainOccurrence {
        paced_train_id: i64,
        occurrence_ref: OccurrenceRef,
    },
```